app/assets/javascripts/activeadmin_addons/all.js

```diff
@@ -479,8 +482,10 @@
           if (isRelation) {
             var value = event.params.data.id;
             var selectedItemsContainer = $("[id='" + prefix + "_selected_values']");
-            var itemName = model + "[" + method + "][]";
+            var itemName = original_name.replace("virtual_", "").replace("_attr", "");
             var itemId = prefix + "_" + value;
+            console.log(itemId);
+            console.log(itemName);
```

Comment:
I think you forgot to remove these, we avoid keeping `console.log`s in the gem code 😄 . Can you remove these lines and the ones in the other files?